text/0000-target-tier-policy.md

```diff
@@ -0,0 +1,222 @@
+- Tier 2 targets must support building on the existing targets used for CI
+  infrastructure.
+- Tier 2 targets must not impose burden on the authors of pull requests, or
+  other developers in the community, to ensure that tests pass for the target.
+  In particular, do not post comments (automated or manual) on a PR that
+  suggests a block on the PR based on tests failing for the target. (A PR
+  author must not break the build of a tier 2 target, but need not ensure the
+  tests pass for a tier 2 target, even if notified that they fail.)
+- The target development team should regularly run the testsuite for the
+  target, and should fix any test failures in a reasonably timely fashion.
+- A tier 2 target may be demoted or removed if it no longer meets these
+  requirements. Any proposal for demotion or removal will be CCed to people who
+  have previously worked on the platform, and will be communicated widely to
+  the Rust community before being dropped from a stable release.
+- All tier 3 requirements apply.
+
+Note: some tier 2 platforms additionally have binaries built to run on them as
+a host (such as `rustc` and `cargo`). Such a platform must meet all the
+requirements above, and must additionally get the compiler and infrastructure
+team to approve the building of host tools.
+
+## Tier 1 target policy
+
+At this tier, the Rust project guarantees that a target builds and passes all
+tests, and will reject patches that fail to build or pass the testsuite on a
+target. We hold tier 1 targets to our highest standard of requirements.
+
+- Any new tier 1 target requires compiler team approval based on these
+  requirements.
+- In addition, the infrastructure team must approve the integration of the
+  target into CI, and the CI-related requirements. This review and approval
+  will typically take place in the PR adding the target to CI.
+- In addition, the release team must approve the long-term viability of the
+  target, and the additional work of supporting the target.
+- Tier 1 targets must have substantial, widespread interest within the
+  developer community, and must serve the ongoing needs of multiple production
+  users of Rust across multiple organizations or projects. A tier 1 target may
+  be demoted or removed if it becomes obsolete or no longer meets this
+  requirement.
+- The target must build and pass tests reliably in CI.
+- Building the target and running the testsuite for the target must not take
+  substantially longer than other targets.
+- If running the testsuite requires additional infrastructure (such as systems
+  running the target), the target development team shall arrange to provide
+  such resources to the Rust project, to the satisfaction and approval of the
+  Rust infrastructure team.
+- Tier 1 targets must provide documentation for the Rust community for how to
+  build and run tests for the platform, using emulation if possible, or
+  dedicated hardware if necessary.
```

Comment:
This is common to all tiers.

Since Tier 3 requirements always apply, perhaps that section should be renamed "Tier 3 target policy (requirements for all supported targets)", and common bullets removed from the tier 2 and tier 3 lists.

Reply:
It isn't quite common to all tiers; tier 3 has a weaker requirement. I've clarified all of the documentation requirements, and since the resulting tier 2 and tier 1 requirements are identical, I've removed the duplicate tier 1 requirement.